src/main/java/fi/otavanopisto/kuntaapi/server/rest/SystemRESTService.java

```diff
@@ -48,6 +49,8 @@
 import fi.otavanopisto.kuntaapi.server.integrations.ServiceSortBy;
 import fi.otavanopisto.kuntaapi.server.integrations.SortDir;
 import fi.otavanopisto.kuntaapi.server.integrations.management.ManagementConsts;
+import fi.otavanopisto.kuntaapi.server.integrations.management.tasks.NewsArticleIdTaskQueue;
+import fi.otavanopisto.kuntaapi.server.integrations.management.tasks.OrganizationNewsArticlesTaskQueue;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'fi.otavanopisto.kuntaapi.server.integrations.management.tasks.OrganizationNewsArticlesTaskQueue'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)